### PR TITLE
[RAINCATCH-942] Development guide

### PR DESCRIPTION
**Pull request by paolobueno**
- [X] Required tooling
- [X] Local setup instructions
- [X] Contribution instructions

- [ ] split cotnributing section

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-development-and-contributing.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[id='{context}-pro-development-and-contributing']
+= Developing and Contributing to the {PROJECT_NAME} Project
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-development-and-contributing.adoc:2]: There will be huge confusion what development means as there are 3 categories of development:
- Core development (contributing)
- RainCatcher step development (plugins,modules)
- Development on the mobile,portal,server application

This confuses even myself and  I think we need to have clear separation between contribution, development of the steps and adding custom functionalities. 

With this in mind `Developing and Contributing to the {PROJECT_NAME} Project` => `Contributing to the {PROJECT_NAME} Project`

**Review by austincunningham — APPROVED** [review]
Still unclear, not sure weather this should go into the contribution section or the development  section

**Comment by wtrocki** [comment]
I think I have got confused over this ticket. We have some instruction for contributions.

**Comment by wtrocki** [comment]
@austincunningham it should be contribution section of the documentation for sure. 
Informations here are standard - we may backport some to CONTRIBUTING.md of the projects itself. 

I'm thinking if we need contribution in the documentation as it may seriously confuse people.
I personally think it's really good to have it documented in docs but we need to make sure that we have clear boundaries in docs.

File: docs/workforce-management-framework/topics/pro-development-and-contributing.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+== Preparing the Development Environment
+
+In order to prepare your local environment to run and develop {PROJECT_NAME}, check the tools in the xref:{context}-ref-development-tools[Tools for Local development] reference and follow the links for installation instructions for the individual tools.
+
+== Retrieve the source code
+
+After cloning the `core` and `angularjs` repositories listed in xref:{context}-identifying-raincatcher-repositories[Identifying {PROJECT_NAME} Repositories] using Git, follow the individual repositories' _README_ files to install code dependencies and start local development.
+
+The individual packages that compose the {PROJECT_NAME} ecossystem are contained in the cloned repositories in a monorepo structure enabled by link:https://lernajs.io/[Lerna.js], see the link:https://github.com/lerna/lerna/[Lerna.js Documentation on GitHub] for more details.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/pro-development-and-contributing.adoc:12]: this sentence could read better,  ``` Lerna.js was used to organise packages in {PROJECT_NAME} into a monorepo structure. For more information on Lerna see ...```

File: docs/workforce-management-framework/topics/pro-development-and-contributing.adoc (hunk 1 above)
Comment (paolobueno) [reply docs/workforce-management-framework/topics/pro-development-and-contributing.adoc:2]: Good insight, I'll try to update and match these categories. :+1:

**Comment by wtrocki** [comment]
@paolobueno This PR is open for long time without any action. 
I'm going to merge that and apply changes myself on master.

**Pull request merged**